thoth-app/src/models/work/mod.rs

```diff
@@ -101,7 +102,17 @@ pub trait DisplayWork {
 
 impl DisplayWork for Work {
     fn onix_endpoint(&self) -> String {
-        format!("{}/onix/{}", THOTH_API, &self.work_id)
+        format!(
+            "{}/specifications/onix_3.0::project_muse/work/{}",
+            THOTH_EXPORT_API, &self.work_id
+        )
+    }
+
+    fn csv_endpoint(&self) -> String {
+        format!(
+            "{}/specifications/csv::thoth/work/{}",
+            THOTH_EXPORT_API, &self.work_id
+        )
     }
```

Comment:
hmm that's odd, I definitely don't get that (I get the whole record)

Reply:
Ah - by coincidence, I had only tested works(/publishers) that didn't have any PDF publication URLs. So `handle_event` was successfully writing out the header and then calling in to `xml_element` which returned nothing. Could this be handled better?

Reply:
Do you get the same using `curl`? e.g. `curl http://localhost:8181/specifications/csv::thoth/work/6248153b-7bbb-45e3-b490-721867cc3d51`

Reply:
Ah wait. It looks like my successful results were actually cached... I'll see what's going on

Reply:
Yes, you're right - it's because of the missing PDFs. Yeah, I want to look into implementing the specification for `Vec<Work>` and `Work` separately, or checking the vector size, so that we can complain if the generation fails when requesting the record of a single work. When querying by publisher it's a separate story...